beetmoverscript/src/beetmoverscript/script.py

```diff
@@ -517,7 +529,7 @@ def generate_system_addons_balrog_manifest(context):
             addon_data = get_addon_data(filepath)
             addon_name = addon_data["name"]
             addon_version = addon_data["version"]
-            addon_url = "{prefix}/{s3_key}".format(prefix=get_bucket_url_prefix(context), s3_key=destinations[0])
+            addon_url = "{prefix}/{s3_key}".format(prefix=get_url_prefix(context), s3_key=destinations[0])
```

Comment:
I kind of want us to rename `s3_key` if it's not s3-specific.

Reply:
Yeah, in this case will likely be a dynamic function. I just haven't really looked into the balrog functions yet. But it's coming :)